custom/inddex/reports/utils.py

```diff
@@ -34,10 +44,11 @@ def _to_context_dict(data_provider):
                 'rows': list(data_provider.rows),
             }
 
-        context = {
+        context = super().report_context
```

Comment:
This pulls in the `report_table_js_options` specified [here](https://github.com/dimagi/commcare-hq/blob/bb07b549c08ebe326580c69dbd9760babb931356/corehq/apps/reports/generic.py#L1106-L1132). This is where the assumption is made that all of the tables in a single inddex report will have the same js options - I'm not pulling them separately for each `data_provider`.